vllm/executor/ray_utils.py

```diff
@@ -101,24 +102,27 @@ def initialize_ray_cluster(
         # Verify that we can use the placement group.
         gpu_bundles = 0
```

Comment:
probably need to change this name!

Reply:
Nice catch! Fixed the variable names from gpu* -> device*